### PR TITLE
Passing all the params through the apply call in lda.get_document_topics()

### PR DESCRIPTION
**Pull request by parthoiiitm**
Related to the issue:
Add per_word_topics parameter to apply call #923 
 
I have applied the modifications in ldamodel.py and interfaces.py, and I am passing the tests in my machine. Please review the code and give feedbacks. Thank you.

### CODE REVIEW

**Comment by parthoiiitm** [comment]
Hi,
   As I have made the changes in the interface.py, I am passing all arguments through call function but it throws error due to the variable chunksize defined in interface.py ( I defined **kwargs after chunksize ). Can you please help where else do I need to make modification to make the code work ?
```
TypeError  Traceback (most recent call last)
<ipython-input-49-9ce45a4637ee> in <module>()
      1 ab = lda.get_document_topics(corpus, per_word_topics=True)
----> 2 for t in ab:
      3     print('new doc')
      4     print(t)

/Users/partho/anaconda/lib/python2.7/site-packages/gensim-0.13.2-py2.7-macosx-10.6-x86_64.egg/gensim/interfaces.pyc in __iter__(self)
    116         if self.chunksize:
    117             for chunk in utils.grouper(self.corpus, self.chunksize):
--> 118                 for transformed in self.obj.__getitem__(chunk, chunksize=None):
    119                     yield transformed
    120         else:

TypeError: __getitem__() got an unexpected keyword argument 'chunksize'
```

**Comment by tmylk** [comment]
Please add a note in CHANGELOG and tests for the new parameters that you have just added.

File: gensim/interfaces.py [hunk 1]
```diff
@@ -123,7 +123,7 @@ def __iter__(self):
 
     def __getitem__(self, docno):
         if hasattr(self.corpus, '__getitem__'):
-           return self.obj[self.corpus[docno]]
```
Comment (tmylk) [inline gensim/interfaces.py]: what is the reason for this change?

File: gensim/models/ldamodel.py [hunk 2]
```diff
@@ -915,8 +915,15 @@ def get_document_topics(self, bow, minimum_probability=None, minimum_phi_value=N
...
         is_corpus, corpus = utils.is_corpus(bow)
+        
+        # Define the extra parameters to be passed in kwargs
+        kwargs = dict(
+            minimum_probability=minimum_probability,
+            minimum_phi_value=minimum_phi_value
+        )
+        
         if is_corpus:
-            return self._apply(corpus)
```
Comment (tmylk) [inline gensim/models/ldamodel.py]: please stay with the existing indent standard

File: gensim/models/ldamodel.py [hunk 3]
```diff
@@ -915,8 +915,15 @@ def get_document_topics(self, bow, minimum_probability=None, minimum_phi_value=N
 
         # if the input vector is a corpus, return a transformed corpus
         is_corpus, corpus = utils.is_corpus(bow)
+        
+        # Define the extra parameters to be passed in kwargs
+        kwargs = dict(
+            minimum_probability=minimum_probability,
```
Comment (tmylk) [inline gensim/models/ldamodel.py]: should per_word_topics be here too?

**Comment by parthoiiitm** [comment]
Can you please advise me on how to proceed to modify the tests.

**Comment by parthoiiitm** [comment]
I have made changes in changelog and put test cases, please review it.

**Comment by parthoiiitm** [comment]
Closing this PR as there's lot of chaos going on in this PR. Will open a fresh one.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,21 +1,26 @@
...
+*  Fix issue #851, In summarizer.py, RunTimeError is raised if single sentence input is provided to avoid ZeroDivionError. (@metalaman, #887)
+* Fix issue [#791](https://github.com/RaRe-Technologies/gensim/issues/791), correct logic for iterating over SimilarityABC interface. ([@MridulS](https://github.com/MridulS), [#839](https://github.com/RaRe-Technologies/gensim/pull/839))
+* Fix RP model loading for large Fortran-order arrays (@piskvorky, [#605](https://github.com/RaRe-Technologies/gensim/issues/938))
+* Remove ShardedCorpus from init because of Theano dependency (@tmylk, [#919](https://github.com/RaRe-Technologies/gensim/pull/919))
+* Documentation improvements ( @dsquareindia & @tmylk, [#914](https://github.com/RaRe-Technologies/gensim/pull/914), [#906](https://github.com/RaRe-Technologies/gensim/pull/906) )
+* Add Annoy memory-mapping example (@harshul1610, [#899](https://github.com/RaRe-Technologies/gensim/pull/899))
+* Passed all the params through the apply call in lda.get_document_topics(), attempt test case to use the per_word_topics through the corpus in test_ldamodel (@parthoiiitm, [#963](https://github.com/RaRe-Technologies/gensim/pull/963))
```
Comment (tmylk) [inline CHANGELOG.md]: remove changes to release 0.13.3 and add the next release 0.13.4. Please add one line to release 0.13.4 describing the change

File: gensim/test/test_ldamodel.py [hunk 5]
```diff
@@ -263,6 +263,21 @@ def testGetDocumentTopics(self):
             self.assertTrue(isinstance(w, int))
             self.assertTrue(isinstance(phi_values, list))
 
+        #Attempt test case to use the get_document_topic function for the corpus
+        doc_topics, word_topics, word_phis = model.get_document_topics(self.corpus, per_word_topics=True)
+
+        for k, v in doc_topics:
+            self.assertTrue(isinstance(k, int))
```
Comment (tmylk) [inline gensim/test/test_ldamodel.py]: please assert the length. it should be for the entire corpus.

**Comment by tmylk** [comment]
See comments and resolve changelog confilcts by pulling in `develop`

File: gensim/test/test_ldamodel.py (hunk 5 above)
Comment (parthoiiitm) [inline gensim/test/test_ldamodel.py]: Should it be like the following??:

```
doc_topics, word_topics, word_phis = model.get_document_topics(self.corpus, per_word_topics=True)
self.assertEqual(len(doc_topics),len(corpus))

for k, v in doc_topics:
            self.assertTrue(isinstance(k, int))
            self.assertTrue(isinstance(v, float))
```

**Comment by tmylk** [comment]
@parthoiiitm  Are you going to open a new clean PR?

**Comment by tmylk** [comment]
Opened #978
